kedro/io/shared_memory_dataset.py

```diff
@@ -10,6 +10,8 @@
 class SharedMemoryDataset(AbstractDataset):
     """``SharedMemoryDataset`` is a wrapper class for a shared MemoryDataset in SyncManager."""
 
+    _EPHEMERAL = True
```

Comment:
For `MemoryDataset` you've only set `self._EPHEMERAL = True` but here we have this line as well as `self._EPHEMERAL = True` in the __init__, why is that?

Reply:
That's a leftover from a previous commit, I'll remove it. Thanks for the catch!